assets/src/helpers/fullStory.ts

```diff
@@ -12,3 +12,12 @@ export const fullStoryInit = (
     }
   }
 }
+
+export const fullStoryEvent = (
+  eventName: string,
+  eventProperties?: { [key: string]: any }
+): void => {
+  FullStory.isInitialized() && FullStory.event(eventName, eventProperties || {})
+
+  return
+}
```

Comment:
suggestion(non-blocking): what do you think about doing this?
I get the benifit of defining our own interface is we can control the translation from our events to their API, but I also like presenting the full API of the event function to the caller as suggested below.
I think this would mean that all the places `{}` was deleted from the call, would need to be reverted.
```suggestion
export const fullStoryEvent = (...args: Parameters<typeof FullStory.event>): void => {
  FullStory.isInitialized() && FullStory.event(...args)
}
```

Reply:
Note: one way or another we still need a `return`, since if `FullStory.isInitialized()` returns `false`, the whole `&&` statement would evaluate to `false` which is incompatible with `void`.

That said, I'm thinking this suggestion over and overall I think I like it.

Reply:
Oh strange, my comment wasn't on the correct lines, sorry about that. It looked right in the editor 😅